Possibly incorrect main in package.json

I followed the instructions at https://svelte.dev/blog/svelte-and-typescript to setup a Svelte + TypeScript project, and used this library **successfully**. But the "svelte validate" step fails with this error:

```
Error: Cannot find module 'svelte-fa' or its corresponding type declarations. (ts)
```

(Note, it can't find the module, which is why it's an error. Other components don't have type declarations, but can be found, and that results in a warning, not an error.)

I *think* this is because `main` in `package.json` is set to a svelte file, which I don't believe is correct.

--- a/package.json
+++ b/package.json
@@ -26,6 +26,7 @@
   "browser": "dist/svelte-fa.js",
   "module": "dist/svelte-fa.mjs",
   "main": "src/fa.svelte",
+  "types": "src/index.d.ts",
   "sideEffects": false,
   "scripts": {
     "build": "rollup -c",

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -1,0 +1,28 @@
+import { SvelteComponent } from "svelte"
+
+declare class Fa extends SvelteComponent {
+  constructor(options: any)
+
+  $$prop_def: {
+    class?: string
+    id?: string
+    style?: string
+
+    icon: { icon: any }
+    fw?: boolean
+    flip?: string
+    pull?: string
+    rotate?: number | string
+    size?: string
+    color?: string
+
+    // Duotone Icons
+    primaryColor?: string
+    secondaryColor?: string
+    primaryOpacity?: number | string
+    secondaryOpacity?: number | string
+    swapOpacity?: boolean
+  }
+}
+
+export default Fa

